data/whispering void/elenchus.txt

```diff
@@ -12,14 +12,14 @@
 # this program. If not, see <https://www.gnu.org/licenses/>.
 
 shipyard "Elenctic Commune Basic"
-	# Simple, old, designs from Hai and humans
+	# Simple, old, designs from Hai and humans.
```

Comment:
Not a full sentence, so the period isn't strictly necessary.
But if we want to be picky:

```suggestion
	# Simple, old designs from Hai and humans.
```